pom.xml

```diff
@@ -24,21 +24,24 @@
 		<cloudevents.version>2.3.0</cloudevents.version>
 		<junit-jupiter.version>5.8.2</junit-jupiter.version>
         <assertj-core.version>3.22.0</assertj-core.version>
-        <jackson.version>2.13.4.2</jackson.version>
+        <jackson.version>2.15.0</jackson.version>
+        <slf4j.version>2.0.7</slf4j.version>
 		<project.github.repository>cdevents/sdk-java</project.github.repository>
 		<nexus.url>https://s01.oss.sonatype.org</nexus.url>
 		<repository.url>git@github.com:${project.github.repository}.git</repository.url>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		<version.clean.plugin>3.1.0</version.clean.plugin>
-		<version.compiler.plugin>3.9.0</version.compiler.plugin>
-		<version.deploy.plugin>3.0.0-M1</version.deploy.plugin>
+		<project.build.outputTimestamp>${git.commit.author.time}</project.build.outputTimestamp>
+		<version.clean.plugin>3.2.0</version.clean.plugin>
+		<version.compiler.plugin>3.11.0</version.compiler.plugin>
+		<version.deploy.plugin>3.1.1</version.deploy.plugin>
+		<version.git.plugin>5.0.0</version.git.plugin>
 		<version.gpg.plugin>1.6</version.gpg.plugin>
-		<version.install.plugin>3.0.0-M1</version.install.plugin>

```

Comment:
A personal question, did maven release a specific version for the Mac M1 chip? Have not seen this version and tried looking it up but could not find an answer for the M1 on the plugin version. Very interesting!

Reply:
This is not a chip related version. M1 usually stands for "milestone". It's just a version tag like GA, FINAL, and so forth.

Reply:
That's fantastic, thanks for sharing! I just haven't seen an M1 version of a plugin before and was excited to learn about it. 😄